bundle/Controller/Admin/DashboardController.php

```diff
@@ -39,6 +40,46 @@ public function indexAction(EntityManagerInterface $entityManager): array
         $repoBroadcast = $entityManager->getRepository(Broadcast::class);
         $repoUsers     = $entityManager->getRepository(User::class);
 
+        $translator = $this->container->get('translator');
```

Comment:
this is forbiden ;-) use autowiring instead.
But even, you don't need the translator here, you can do this in twig
Something like

```twig
{{ ("dashboard.users.status."~item.status)|trans }}
```

Also the key should be more generic, `users.status.*`